entrypoint.sh

```diff
@@ -2,6 +2,8 @@
 
 cd "${GITHUB_WORKSPACE}" || exit 1
 
+git config --global --add safe.directory $GITHUB_WORKSPACE || exit 1
```

Comment:
how can I write this in yaml file to prevent the detekt failure which is happening due to reviewDog?